libs/vm-modules/src/math/tensor/tensor.cpp

```diff
@@ -85,6 +90,7 @@ void VMTensor::Bind(Module &module, bool const enable_experimental)
 
   auto interface =
       module.CreateClassType<VMTensor>("Tensor")
+          .CreateConstructor(&VMTensor::EmptyConstructor)
```

Comment:
you still need to charge for doing that!

Also you should make this an experimental binding

Reply:
ok, got it